app/View/Components/AppLayout.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\View\Components;
+
+use Illuminate\View\Component;
+
+class AppLayout extends Component
+{
+    /**
+     * Get the view / contents that represents the component.
+     *
+     * @return \Illuminate\View\View
+     */
+    public function render()
+    {
+        return view('layouts.app');
+    }
+}
```

Comment:
You should be able to remove this and have the component still work as it'll register itself as an anonymous component.

Ref: https://laravel.com/docs/8.x/blade#anonymous-components

Reply:
912e711